src/lib.rs

```diff
@@ -181,6 +184,26 @@ impl<'a> fmt::Display for Record<'a> {
     }
 }
 
+/// Write Record Extension on io::Write
+pub trait WriteRecordExt: io::Write {
+    
+    /// write a record to a io::Write implementor
+    ///
+    /// returns the length of the written record
+    fn write_record(&mut self, record: Record) -> Result<usize>;
+}
+
+impl WriteRecordExt for File {
```

Comment:
I believe it should be implemented for `T: io::Write` which will cover every `io::Write` implementor including `fs::File`